py/setup.py

```diff
@@ -32,6 +32,12 @@
     'description': 'Python bindings for Selenium',
     'long_description': open(join(abspath(dirname(__file__)), "README.rst")).read(),
     'url': 'https://github.com/SeleniumHQ/selenium/',
+    'project_urls': {
+        'Bug Tracker': 'https://github.com/SeleniumHQ/selenium/issues',
+        'Changes': 'https://github.com/SeleniumHQ/selenium/blob/trunk/py/CHANGES',
```

Comment:
is this better pointing at: `https://github.com/SeleniumHQ/selenium/blob/trunk/py/CHANGES` specifically?

Reply:
@symonk That's the same URL? Is there a different one you'd prefer?

Reply:
apologies missed the `/py/`